orderbook/src/main.rs

```diff
@@ -27,10 +34,24 @@ pub async fn orderbook_maintenance(orderbook: Arc<OrderBook>) -> ! {
 }
 #[tokio::main]
 async fn main() {
+    // Todo: merge valentins PR to get it from env
+    const NODE_URL: &str = "http://localhost:8545";
+
+    let http = Http::new(NODE_URL).expect("Node url invalid, or service not available");
+    let web3 = Web3::new(http);
+
+    let chain_id = web3.eth().chain_id().await.expect("Could not get chainId");
```

Comment:
Probably, we want to run here:
```
wait_for_node(&web3).await?;
```

But I was not able to export from the other crate in a proper way.

Reply:
I think it's ok to assume the RPC endpoint is running when we start the API.